packageclients/pkg/packageclient/image_utils.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2021 VMware, Inc. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package tkgpackageclient
+package packageclient
```

Comment:
Usually, package names are not in camel case. Also, to be consistent with others, shouldn't this be `packageclient`?

Reply:
This is not camelcase. And I've already renamed it to ```packageclient```, so it is correct.

Reply:
I might have been seeing the old version. When I left the review comment, I saw the package name as `packageClient` (camel case). But I see now it is how I had suggested, `packageclient`.